frontend/resourceadm/components/AccessListMembers/AccessListMembersTable.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Table } from '@digdir/design-system-react';
+import { AccessListMember } from 'app-shared/types/ResourceAdm';
+import { StudioButton } from '@studio/components';
+import classes from './AccessListMembers.module.css';
+
+interface AccessListMembersTableProps {
+  listItems: AccessListMember[];
+  buttonNode: React.ReactNode;
+  isHeaderHidden?: boolean;
+  disableButtonFn?: (member: AccessListMember) => boolean;
+  onButtonClick: (member: AccessListMember) => void;
+}
+
+export const AccessListMembersTable = ({
+  listItems,
+  buttonNode,
+  isHeaderHidden,
+  disableButtonFn,
+  onButtonClick,
+}: AccessListMembersTableProps): React.ReactNode => {
```

Comment:
Then let's use React.JSX.Element, since the JSX namespace is deprecated: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/266eae5148c535e6b41fe5d0adb2ad23f302bc8a/types/react/index.d.ts#L3182